handler.ts

```diff
@@ -158,8 +180,24 @@ module.exports.reverse = bugsnagHandler(
     context: null,
     callback: ServerlessCallbackFunction
   ): Promise<void> => {
-    const response = await makePeliasRequests(event, 'reverse')
+    const geocoderResponse = await getPrimaryGeocoder().reverse(
+      convertQSPToGeocoderArgs(event.queryStringParameters)
+    )
 
-    callback(null, response)
+    callback(null, {
+      body: JSON.stringify(geocoderResponse),
+      /*
+        The third "standard" CORS header, Access-Control-Allow-Methods is not included here
+        following reccomendations in https://www.serverless.com/blog/cors-api-gateway-survival-guide/
+
+        This header is handled within AWS API Gateway, via the serverless CORS setting.
+        */
+      headers: {
+        'Access-Control-Allow-Credentials': 'true',
+        'Access-Control-Allow-Origin': '*',
+        'Content-Type': 'application/json'
+      },
+      statusCode: 200
+    })
```

Comment:
I just realized, why is `reverse` the only endpoint that gets manual cors headers? Wasn't this handled via the `serverless.yml`?

Reply:
It is the same headers that are included in the response from makeGeocoderRequests function, which is what is used by the other two endpoints. I was just following that pattern.

Reply:
Thanks for looking into that! Sounds excellent